src/eko/harmonics/__init__.py

```diff
@@ -98,3 +100,63 @@ def get_smx(n, max_weight=5):
     if max_weight >= 5:
         smx[4] = Sm5(n)
     return smx
+
+
+@nb.njit(cache=True)
+def get_s3x(n, sx, smx):
```

Comment:
Usually, is more pythonic to call things (mainly properties, but even functions) without prepended `get_`.

Thing like `srx(...)` it's just the object, that simply depends on some parameters.